### PR TITLE
Add read-only bots panel

### PR DESCRIPTION
**Pull request by hyperobject**
Adds RPC calls and UI for a new "Bots" tab in the chat info pane, which contains a list of bot/restricted bot users in the current channel, as well as a list of featured bots that are available to add. This will need a style pass to fix some weird scroll bar stuff, but it should be working for the most part.

### CODE REVIEW

File: shared/actions/bots.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      logger.info(err.message)
+    }
+    return
+  }
+}
+
+const searchFeaturedBots = async (state: Container.TypedState, action: BotsGen.SearchFeaturedBotsPayload) => {
+  const {limit, offset, query} = action.payload
+
+  if (state.chat2.featuredBotsMap.has(query)) {
```
Comment (joshblum) [inline shared/actions/bots.tsx]: search may return more than 1 result even in an exact match on one bot, might not want to cache this this way

File: shared/actions/bots.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    // don't refresh featured bot if it's in the store
+    return
+  }
+  try {
+    const {bots} = await RPCTypes.featuredBotSearchRpcPromise({limit: limit ?? 1, offset: offset ?? 0, query})
```
Comment (joshblum) [inline shared/actions/bots.tsx]: don't think you should limit to 1 here

File: shared/chat/conversation/info-panel/bot.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      )}
+    </Kb.Box2>
+  )
+
+  const usernameDisplay = (
+    <Kb.Box2 direction="horizontal" alignSelf="flex-start">
+      <Kb.Text type="BodySmallSemibold" style={{color: Styles.globalColors.black}}>
+        {botAlias || botUsername}
+      </Kb.Text>
+      <Kb.Text type="BodySmall">&nbsp;• by @{ownerTeam ?? ownerUser ?? botUsername}</Kb.Text>
```
Comment (joshblum) [inline shared/chat/conversation/info-panel/bot.tsx]: this shouldn't default to `botUsername`, one of `ownerTeam` or `ownerUser` will always be set. I think this is supposed to be a link to the team/user page and not just text

File: shared/actions/bots.tsx (hunk 1 above)
Comment (hyperobject) [reply shared/actions/bots.tsx]: gotcha, i'll take that out

File: shared/chat/conversation/info-panel/bot.tsx (hunk 3 above)
Comment (hyperobject) [reply shared/chat/conversation/info-panel/bot.tsx]: I defaulted to botUsername in cases where there's a restricted bot that isn't in our database (like dr jenkins for example). adding a link shouldn't be too tricky

File: shared/actions/bots.tsx (hunk 2 above)
Comment (hyperobject) [reply shared/actions/bots.tsx]: that's just the default limit, is there another sensible default?

File: shared/chat/conversation/info-panel/bot.tsx (hunk 3 above)
Comment (joshblum) [reply shared/chat/conversation/info-panel/bot.tsx]: ah nice makes sense

File: shared/actions/bots.tsx (hunk 2 above)
Comment (joshblum) [reply shared/actions/bots.tsx]: 10?

**Comment by mmaxim** [comment]
Can we turn off the server feature flag now?

**Comment by mmaxim** [comment]
At least on a per team basis

**Comment by joshblum** [comment]
> Can we turn off the server feature flag now?

Sure i'll restrict it to admin only

File: shared/chat/conversation/info-panel/bot.tsx (hunk 3 above)
Comment (hyperobject) [reply shared/chat/conversation/info-panel/bot.tsx]: Getting the team to show a popup is a larger process than expected, so I'll throw that into another ticket

**Pull request merged**